docs/content/en/docs/getting-started/install/_index.md

```diff
@@ -5,36 +5,38 @@ description: >
   Steps for setting up the Admin Machine
 ---
 
+{{% alert title="Warning" color="warning" %}}
+The Administrative machine should not be ephemeral. Artifacts created during cluster operations will be stored on the Administrative machine. Losing this machine could result in loss of cluster information.
```

Comment:
We can be clearer here. 
"The Administrative machine (Admin machine) is required to run cluster lifecycle operations, but EKS Anywhere clusters do not require a continuously running Admin machine to function. During cluster creation, critical cluster artifacts including the kubeconfig file, SSH keys, and the full cluster specification yaml are saved to the Admin machine. These files are required when running any subsequent cluster lifecycle operations. For this reason, it is recommended to save a backup of these files and to use the same Admin machine for all subsequent cluster lifecycle operations."

Reply:
Sounds good, using updated verbiage.